### ISSUE TITLE
Gene Search Tool and downstream issues encountered

### ISSUE THREAD

**Issue by sdash-github** [PeanutBase/jekyll-peanutbase#50]
I found this while developing a user guide; now work in progress at  https://dev.peanutbase.org/userguides/genefamily/geneToFamily.html  

**Go to gene family starting from a gene**
1. **Using Gene Search Tool**
   * PeanutBase Home page (use left Tools panel or Tools dropdown in menubar) to [Gene Search Tool](/tools/gene-keyword    -search.html){:_blank}.
   * Use the Identifier/Description SEARCH boxex to type your gene name, id, or even partial id.
`Arahy.H916D0 fails but H916D0 works while in ArachisMine both works. So confusing for user.` 
`More confusing because the example  shows a Soy pattern that fails, e.g. Glyma.13G357700.  `
`We need a Arahy example for PB.`
  * If found the Result below will list all the genes and other features that has this id/name.
  * The 'GENE FAMILY ASSIGNMENTS' column will list the link to the Gene Family report page which has a link on the right panel named ,'View on the LIS Funnotate app'. 
`It appears under 'External Links' here while in the  alternate ArachisMines search it appears under ‘LIS Linkouts’ which is more appropriate.`
* This links to the Phylogram for the gene family and shows the Taxa  composition piechart and the tree diagram below with the searched gene highlighted.  
`This link (https://funnotate.legumeinfo.org/?family=legfed_v1_0.L_6RNLR0) is unproductive as it goes to Funnotate but shows  perpetual machine loading.`
* (This page is an interface to the ArachisMine from PeanutBase)

**Comment by adf-ncgr** [comment]
I've created [an issue](https://github.com/legumeinfo/web-components/issues/277) for allowing customization of the examples. I can't reproduce the issue with funnotate not displaying the tree for the linked family. I think we ought to use a gnm2.ann2 gene as the example, and just avoid the mess with "H916D0 vs Arahy.H916D0" which is more of a data issue than anything else. Let me know if there's something else that needs resolving here.

**Comment by sdash-github** [comment]
This link from the gene family report page is still perpetually loading. It is also labeled under External Links but should be LIS linkout.
https://funnotate.legumeinfo.org/?family=legfed_v1_0.L_6RNLR0  

<img width="1228" alt="Screen Shot 2023-11-24 at 6 29 14 PM" src="https://github.com/PeanutBase/jekyll-peanutbase/assets/3673723/df9143eb-6569-4bad-a7ee-a76f67b05ba4">

**Comment by adf-ncgr** [comment]
The URL you provided is still working on my end. Do you see any errors in the browser console?

**Comment by sdash-github** [comment]
**1. Console error:**
`
Uncaught SyntaxError: Identifier 'LINKOUTS_BASE_URL' has already been declared (at phylogram.js:1:1)
sockjs.min.js:sourcemap:2 WebSocket connection to 'wss://funnotate.legumeinfo.org/__sockjs__/n=HXwiKZfOwB8vargttu/927/spt0mg1d/websocket' failed: 
n.exports @ sockjs.min.js:sourcemap:2
shiny-server-client.min.js:1 Fri Nov 24 2023 12:22:44 GMT-0600 (Central Standard Time) [INF]: Connection opened. https://funnotate.legumeinfo.org/?family=legfed_v1_0.L_6RNLR0
shiny-server-client.min.js:1 Fri Nov 24 2023 12:22:44 GMT-0600 (Central Standard Time) [DBG]: Open channel 0
funnotate.legumeinfo.org/:1 Uncaught (in promise) Error: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
`
2. When I opened the URL incognito it worked. So something at my end??

**Comment by adf-ncgr** [comment]
Yes, I'm not exactly sure how to interpret the error, but I think your browser must have gotten into a problematic state. Maybe try clearing the cache?

**Comment by svengato** [comment]
> This link (https://funnotate.legumeinfo.org/?family=legfed_v1_0.L_6RNLR0) is unproductive as it goes to Funnotate but shows  perpetual machine loading.

It works for me too, I get a phylotree in a few seconds.
